src/methods/method_overloading/medium/q3/ThreeStrings.java

```diff
@@ -0,0 +1,28 @@
+package methods.method_overloading.medium.q3;
+
+/*
+Overload a Java method so that it can take two strings or can take three strings and concatenate them (combines all the
```

Comment:
Change question text to:
`Overload a Java method called concatenate so that it can take two or three strings and concatenate them (combines all the strings into one large string). The main method should print two calls to the method, one with two inputs and one with three.`